examples/meteringlab/c2/cassandra-stress.yaml

```diff
@@ -0,0 +1,40 @@
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cassandra-stress-0
+  namespace: default
+  labels:
+    app: cassandra-stress
+spec:
+  template:
+    spec:
+      containers:
+      - name: cassandra-stress
+        image: scylladb/scylla:4.0.0
```

Comment:
you need cassandra-stress which supports connecting through sni proxy, this one doesn't. Afaik QA is using scylla-bench.